### PR TITLE
nftables getting-started guide

### PR DESCRIPTION
**Pull request by caseydavenport**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: sidebars-calico.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ module.exports = {
             'getting-started/kubernetes/requirements',
             'getting-started/kubernetes/community-tested',
             'getting-started/kubernetes/quickstart',
+            'getting-started/kubernetes/nftables',
```
Comment (ctauchen) [inline sidebars-calico.js]: There's a larger conversation about how and where we document dataplane guides like this. But for now it makes most sense to group this further down next to the VPP guide.

File: calico/getting-started/kubernetes/nftables.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
+---
+description: Install Calico using the tech-preview nftables data plane.
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/nftables.mdx]: I can see this easily being missed when it goes GA. Can we scrap the tech preview language here?

File: calico/getting-started/kubernetes/nftables.mdx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+---
+
+# Calico nftables data plane
+
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/nftables.mdx:9]:
```suggestion
:::note

This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.

:::
```
We're using this boilerplate for page-level tech preview notices.

File: calico/getting-started/kubernetes/nftables.mdx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+- {{prodname}} can manage `cali` and `tunl` interfaces on the host
+
+  If NetworkManager is present on the host, see [Configure NetworkManager](../../operations/troubleshoot/troubleshooting.mdx#configure-networkmanager).
+
+:::note
+
+{{prodname}} nftables dataplane is currently in technical preview and may not be suitable for production use. Please report any issues you experience
+while using this feature.
+
+:::
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/nftables.mdx]: Remove, in favor of boilerplate above.

File: calico/getting-started/kubernetes/nftables.mdx [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+   After installing kubeadm, do not power down or restart
+   the host. Instead, continue directly to the next step.
+
+   :::
+
+1. As a regular user with sudo privileges, open a terminal on the host that you installed kubeadm on.
+
+1. Create a kubeadm configuration file that enables the nftables kube-proxy mode.
+
+   ```
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/nftables.mdx]:
```suggestion
   ```bash
```
Globally, for syntax highlighting.

**Review by ctauchen — COMMENTED** [review]
A few suggestions for you. This is a good doc to get folks up and running with the new dataplane. It would be worth discussing how we ought to be presenting dataplane information generally in the docs. For both eBPF and VPP, we're relying on bespoke guides, not unlike this one, that were written to present the tech preview feature. But we should see how we can integrate these procedures as part of a standard installation procedure.

**Comment by caseydavenport** [comment]
@ctauchen thanks for the review! I think I've addressed your comments.

> But we should see how we can integrate these procedures as part of a standard installation procedure.

Definitely agree, but I think it's larger than this PR.

It might be nice to have data plane selection more seamless integrated into the other install docs rather than tucked away in its own set. The only concern I see is complicating what would otherwise be a pretty straightforward and opinionated guide with information that isn't relevant to every user.

**Comment by ctauchen** [comment]
> It might be nice to have data plane selection more seamless integrated into the other install docs rather than tucked away in its own set. The only concern I see is complicating what would otherwise be a pretty straightforward and opinionated guide with information that isn't relevant to every user.

Yes, I think it's all worth doing, but it will have to be part of a much larger effort. 

For now, do you want to shop this doc around a bit, or are you happy to merge? @caseydavenport

**Comment by caseydavenport** [comment]
I think I'm good to merge this :+1:

**Pull request merged**